script.js

```diff
@@ -0,0 +1,66 @@
+let startBtn = document.querySelector("#startButton");
+startBtn.addEventListener("click", playGame);
+//ash says we can use this as a global object
+let gameState = ["", "", "", "", "", "", "", "", ""];
+
+function decideTurn(currentPlayer) {
+  if (currentPlayer === "o") {
+    return "x";
+  }
+  return "o";
+}
+function addToScreen(tile, currentPlayer) {
+  tile.innerHTML = currentPlayer;
+}
+
+function alterGameState(gameState, clicked, currentPlayer) {
+  if (gameState[clicked] === "") {
+    gameState[clicked] = currentPlayer;
+    return true;
+  }
+  return false;
+}
+
+function playGame() {
+  // this determines turn order
+  // this variable holds alternating player symbols
+  let currentPlayer = "x";
+  const tiles = document.querySelectorAll(".tile");
+  startBtn.removeEventListener("click", playGame, false);
+  startBtn.style.opacity = 0.5;
+  tiles.forEach(function (tile) {
+    tile.addEventListener("click", () => {
+      //gets index value of tile from data attribute
+      let clicked = tile.dataset.index;
+      //assigns player marker based on currentPlayer value
+      //this is a boolean to regulate whether a player can click a tile.
+      let notYetClicked = alterGameState(gameState, clicked, currentPlayer);
+      //as long as tile has not been clicked,
+      //will add mark to gameState array
+      //and increment playerScore for next round
+      if (notYetClicked) {
+        addToScreen(tile, currentPlayer);
+        currentPlayer = decideTurn(currentPlayer);
+      }
+    });
+  });
+}
+
+function showModal(winner) {
+  let close = document.querySelector(".close");
+  let modal = document.querySelector(".modal");
+  let displayWinner = document.querySelector("#displayWinner");
+
+  modal.style.display = "block";
+  if (winner.includes("1")) {
```

Comment:
We need the string from the winner variable for Story 7 as well but have passed in the entire string. I'm not sure our way or your way is the most concise way of doing it? e.g. would it be worth make the string held by winner less lengthy so it's more usable as is rather than having to make these workarounds?

Reply:
same goes for line 58